user-script-register/README.md

```diff
@@ -0,0 +1,19 @@
+# User script registration
+
+This extension demonstrates the [`browser.userScripts.register()`](https://wiki.developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/userScripts/Register) API.  
+
+The extension includes an [API script](https://wiki.developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/user_scripts) (`customUserScriptAPIs.js`) that registers a script that enables user scripts to make use of `browser.storage.local`. 
+
+To enable a user script to be specified and registered, this extension includes a sidebar action. The sidebar enables you to define the following properties that control the execution of a registered script, with default values provided:
+
+* the host pattern `*://*.org/*`, which loads the script into any HTTP or HTTPS pages on a `.org` domain.
+* script code that replaces the content of the pattern matched page with the message "This page has been eaten". The script also uses the API script stubs to save and recall the URL of each page "eaten". Information on the last and current "eaten" page is then included in the "eaten" message.
+
+All other properties are take their default value.
```

Comment:
```suggestion
All other properties are given their default value.
```

Reply:
Perhaps "use" would be more accurate, as the code doesn't do anything to give the other  properties their default values